src/design-system/atoms/Input.tsx

```diff
@@ -2,13 +2,19 @@ import { InputHTMLAttributes } from "react";
 import styled from "styled-components";
 
 const StyledInput = styled.input`
-   color:orange;
-`;
+  color: white;
+  border: 2px solid gray;
+  background-color: #d7b6d7;
```

Comment:
@ianleeduckworth this purple color should be white.

Reply:
@hopeduckworth That makes sense. However, I'm noticing that the text color is also white which is (obviously) an accessibility issue. Should the text color maybe be changed? I can make it whatever you want, just let me know! 🚀

Reply:
Yes please!  Let's use black for now.  We may change it later but we'll keep it black for now.

Reply:
Ok great! I've updated the colors and they have good contrast now. Thanks.